toolchain/check/handle_modifier.cpp

```diff
@@ -34,6 +34,8 @@ static auto DiagnoseNotAllowedWith(Context& context, Parse::NodeId first_node,
       .Emit();
 }
 
+// Handles the main modifier keyword. If valid, adds it to the modifier set and
```

Comment:
What's the _main_ modifier keyword?

Reply:
I've tried to give a couple more examples of what this handles for modifiers with multiple keywords.